main.tf

```diff
@@ -0,0 +1,116 @@
+terraform {
```

Comment:
[q] have you run [`terraform fmt`](https://spacelift.io/blog/terraform-fmt#what-is-terraform-fmt) to ensure proper formatting? When adding github actions for we can include a step that ensure correct formatting.